config.ini

```diff
@@ -1,22 +1,22 @@
 [PATHS]
-input_path = D:\Programming\Facultate\Licenta\Input
-output_path = D:\Programming\Facultate\Licenta\Output
+input_dir = /storage/maria/The real D/Programming/Facultate/Licenta 1.0/input
+output_dir = /storage/maria/The real D/Programming/Facultate/Licenta 1.0/output
```

Comment:
Why are the dirs set here and below also ? 
 OUTPUT_DIR = os.path.join(ROOT_DIR, 'output')

Reply:
config.ini file is automatically generated when calling the project, but only if it's missing (or non-valid, but not implemented yet)
The path can be changed directly in the config.ini file if the user wants the input or output path to be somewhere else, but I needed a default output path which will automatically save output in the set directory, if the user doesn't want to change the location. ( I'll add validation of paths later, so that if the user wants to change either input or output path in the configuration file and writes an non-existent one, it will be recreated with the default settings ).
I'll stop commiting it ( noted ;) )